puzzles/models.py

```diff
@@ -175,6 +175,22 @@ def can_delete(self):
     def correct_answers(self):
         return [answer.text for answer in self.guesses.filter(status=Answer.CORRECT)]
 
+    def create_field_url_map(self):
+        """
+        Creates dictionary mapping text -> url for all relevant urls if they exist.
+        """
+        field_url_map = {}
+        field_url_map["Puzzle"] = self.url
+        if self.chat_room:
+            if self.chat_room.text_channel_url:
+                field_url_map["Text"] = self.chat_room.text_channel_url
```

Comment:
I feel like "Text" could be a little clearer but I don't actually have a good alternative. "Text chat"? "Text channel"? "Chat" (sounds confusing w/ voice chat but if there's a separate Voice button maybe it won't be confusing)? Not sure

Reply:
Renamed to "Text Channel" and "Voice Channel"